public-api/src/main/java/org/pmiops/workbench/cdr/dao/QuestionConceptDao.java

```diff
@@ -7,11 +7,20 @@
 
 public interface QuestionConceptDao extends CrudRepository<QuestionConcept, Long> {
 
-    @Query(nativeQuery=true, value="SELECT c.concept_id,c.concept_name,c.domain_id,c.vocabulary_id,c.concept_code,c.count_value,c.prevalence from\n" +
-            "concept c \n" +
-            "join achilles_results ar on ar.stratum_2=c.concept_id\n" +
-            "where ar.stratum_1=?1 and ar.analysis_id=3110\n" +
-            "group by c.concept_id,c.concept_name,c.domain_id,c.vocabulary_id,c.concept_code,c.count_value,c.prevalence,ar.stratum_5 \n" +
-            "order by CAST(ar.stratum_5 AS UNSIGNED) asc")
+    @Query(nativeQuery=true, value="SELECT c.concept_id,c.concept_name,c.domain_id,c.vocabulary_id,c.concept_code,c.count_value,c.prevalence,\n" +
+            "(select count(distinct question_concept_id) from survey_question_map where path=sqm.id and is_main=0) as source_count_value from concept c \n" +
```

Comment:
I'm a bit lost, does a survey always have subquestions?

Reply:
Not always only single question in a single survey.

Reply:
Per slack discussion, consider whether we could use an `@Formula` annotation instead of overloading the `source_count_value` here.

https://stackoverflow.com/a/2986354